pkg/cmd/attestation/verification/attestation_test.go

```diff
@@ -12,11 +14,32 @@ import (
 )
 
 func TestLoadBundlesFromJSONLinesFile(t *testing.T) {
-	path := "../test/data/sigstore-js-2.1.0_with_2_bundles.jsonl"
-	attestations, err := loadBundlesFromJSONLinesFile(path)
+	t.Run("with original file", func(t *testing.T) {
+		path := "../test/data/sigstore-js-2.1.0_with_2_bundles.jsonl"
+		attestations, err := loadBundlesFromJSONLinesFile(path)
+		require.NoError(t, err)
+		require.Len(t, attestations, 2)
+	})
 
-	require.NoError(t, err)
-	require.Len(t, attestations, 2)
+	t.Run("with extra lines", func(t *testing.T) {
+		// Create a temporary file with extra lines
+		tempDir := t.TempDir()
+		tempFile := filepath.Join(tempDir, "test_with_extra_lines.jsonl")
+
+		originalContent, err := os.ReadFile("../test/data/sigstore-js-2.1.0_with_2_bundles.jsonl")
+		require.NoError(t, err)
+
+		extraLines := []byte("\n\n{\"invalid\": \"json\"}\n\n")
```

Comment:
It looks like this test is failing because of the including test JSON object on this line. The code fails because it expects the content of the JSON to match a certain schema.

Since we just need to test that the code can parse multiple JSON objects in a file, I think we can remove the test JSON object and instead just add the extra newlines you have included.

```suggestion
		extraLines := []byte("\n\n")
```

Reply:
> It looks like this test is failing because of the including test JSON object on this line. The code fails because it expects the content of the JSON to match a certain schema.
> 
> Since we just need to test that the code can parse multiple JSON objects in a file, I think we can remove the test JSON object and instead just add the extra newlines you have included.

Okay got it, will make this change along with the formatting